specification/dnsresolver/resource-manager/Microsoft.Network/stable/2022-06-21/dnsresolver.json

```diff
@@ -0,0 +1,2675 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DnsResolverManagementClient",
+    "description": "The DNS Resolver Management Client.",
+    "version": "2022-06-21"
```

Comment:
>2022-06-21 [](http://example.com/codeflow?start=16&length=10)

 please follow this: The first commit needs to be the previous api version and the new changes should only be added in the subsequent commits. This is explained in detail here (If using Open API which is recommended)  [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version) or here(For manual process) :  [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version) . Please remove the "ARMChangesRequested" tag once you have fixed this issue.

Reply:
Added two more commits: one commit with the previous API version and one with the new API version.

https://github.com/Azure/azure-rest-api-specs/compare/7ce9c574385df538883e418b6e0c98a4bd114e83..bef8302b871c461efac6833036b093842bfba927

Reply:
@rkmanda : I don't have access to remove the label. Can you please do it for me if I have addressed this comment appropriately?